src/d3a/d3a_core/cli.py

```diff
@@ -61,9 +64,9 @@ def main(log_level):
 @main.command()
 @click.option('-d', '--duration', type=IntervalType('D:H'), default="1d", show_default=True,
               help="Duration of simulation")
-@click.option('-t', '--tick-length', type=IntervalType('M:S'), default="1s", show_default=True,
+@click.option('-t', '--tick_length', type=IntervalType('M:S'), default="1s", show_default=True,
```

Comment:
I would stick to use a dash for the CLI parameters, and I would also fix the CLI parameters that use underscore. It is somehow a standard in most command lines to never use _ and always use - for word separation.

Reply:
Ahh, i see, and it gets mapped from "-" to "_" automatically by click?

Reply:
Yes, you can also check in the changes before the pause-after parameter, which is mapped to pause_after variable